API/builder/targets/artik053.py

```diff
@@ -122,7 +122,7 @@ def _build_iotjs(self, profile, extra_flags):
 
     def _append_testfiles(self):
         '''
-        Add test files to the ROMFS file of NuttX.
+        Add test files to the ROMFS file of TizenRT.
```

Comment:
This comment is only true for the NuttX target (where the default ROMFS image is overwritten). Instead, I would add the following comment:
Add test files to the content folder of TizenRT.

Reply:
Fixed.